sensibo/Utils.js

```diff
@@ -1,33 +1,46 @@
-// FIXME: This files location should move...
-// TODO: move unifed.js funcs in to here
+// eslint-disable-next-line no-unused-vars
+const SensiboACPlatform = require('../sensibo/SensiboACPlatform')
+// eslint-disable-next-line no-unused-vars
+const SensiboAccessory = require ('../homekit/SensiboAccessory')
+const unified = require('../sensibo/unified')
+
+// FIXME: This file should be moved to a different location...
+// TODO: move unified.js functions here
+
+/**
+ * @param {SensiboAccessory} device
+ * @param {SensiboACPlatform} platform
+ */
 module.exports = (device, platform) => {
 	const Characteristic = platform.api.hap.Characteristic
-	const log = platform.log
+	const easyDebugInfo = platform.easyDebugInfo
+	const easyDebugError = platform.easyDebugError
+	const easyDebugWarning = platform.easyDebugWarning
 
 	return {
 
 		/**
 		 * Convert degrees F to degrees C
 		 * @param  {Number} degreesF The degrees in F to convert
-		 * @returns {Number}         The degrees in C
+		 * @return {Number} The degrees in C
 		 */
 		toCelsius: (degreesF) => {
-			const degreesC = (degreesF - 32) / 1.8
+			const degreesC = unified.toCelsius(degreesF)
```

Comment:
I'm deprecating unified.toCelsius with the plan to move all usage to Utils.toCelsius

In the long run I want to remove unified.js all together

Reply:
Sure, let's fix this in the next version.

Reply:
Shouldn't / couldn't it just be done now?

```suggestion
			const degreesC = (degreesF - 32) / 1.8
```

Reply:
We could, but if we only made this change, we would still have code that uses unified.toCelsius within unified itself and we would then have to code paths doing the exact same. Not a great practice IMHO. If we want to deprecate it, we need to have no code using it. Since that's a bit bigger deal, I'd rather deal with it after we finish with this version.